package.json

```
@@ -25,6 +25,7 @@
   "devDependencies": {
     "@microsoft/api-documenter": "^7.19.9",
     "@microsoft/api-extractor": "^7.29.5",
+    "@siteimprove/alfa-cli": "workspace:^0.62.0",
```

```suggestion
    "@siteimprove/alfa-cli": "workspace:^0.62.3",
```
Version numbers differ between this repo and the main Alfa one, this is intended.
And locally, I messed up trying to build the first release with that repo, so that ended up being 0.62.3 rather than 0.62.0 🙈